src/UserSpaceInstrumentation/InjectLibraryInTracee.cpp

```diff
@@ -224,7 +223,7 @@ ErrorMessageOr<uint64_t> FindFunctionAddressWithFallback(pid_t pid, std::string_
 [[nodiscard]] ErrorMessageOr<void> DlcloseInTracee(pid_t pid, void* handle) {
   // Figure out address of dlclose.
   OUTCOME_TRY(address_dlclose,
-              FindFunctionAddressWithFallback(pid, "dlclose", "libdl", "__libc_dlclose", "libc"));
+              FindFunctionAddressWithFallback(pid, "dlclose", "libdl.so.2", "__libc_dlclose", "libc.so.6"));
```

Comment:
nitpick - maybe introduce constanst for these? kLibcSoname, kLibdlSoname? kDlopenInLibdl/kDlopenInLibc

Reply:
done